action.yml

```diff
@@ -197,6 +201,22 @@ runs:
           echo "::endgroup::"
         fi
 
+    - name: Copy documentation
+      shell: bash
+      run: |-
+        echo "::group::Copying documentation"
+        cp /tmp/${{ inputs.sdk-name }}/docs/* ${{ inputs.documentation-dir }}/compute-${{ inputs.sdk-name }} -r
```

Comment:
I am not sure if the folder name will always start with the `compute-` prefix... maybe adding a check before or include the whole path into the `documentation-dir`.